src/components/ProfileDropdown.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles, makeStyles } from '@material-ui/core/styles';
+import {
+  Avatar,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import NewIcon from '@material-ui/icons/NewReleases';
+import BookmarkIcon from '@material-ui/icons/Bookmark';
+import DoneIcon from '@material-ui/icons/Done';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+
+// hacky way to override Menu style
+const StyledMenu = withStyles((theme) => ({
+  paper: {
+    border: '1px solid black',
+    width: '350px',
+    //background: theme.palette.background.default,
+  },
+  // placing dropdown menu below the avatar
+}))((props) => (
+  <Menu
+    elevation={0}
+    getContentAnchorEl={null}
+    anchorOrigin={{
+      vertical: 'bottom',
+      horizontal: 'center',
+    }}
+    transformOrigin={{
+      vertical: 'top',
+      horizontal: 'center',
+    }}
+    {...props}
+  />
+));
+
+const useStyles = makeStyles((theme) => ({
+  root: {
```

Comment:
This is confusing. I thought this was referring to the root of the overall component but it looks like it's the root of the account header section. Maybe rename to `gridRoot`?

Reply:
Ok name changed